packages/manager/apps/dedicated/client/app/shell.js

```diff
@@ -0,0 +1,19 @@
+let shellClient;
+
+export const setShellClient = (client) => {
+  shellClient = client;
+
+  // set callbacks
+  // set on locale change
+  shellClient.i18n.onLocaleChange(() => {
+    window.top.location.reload();
+  });
+
+  return shellClient;
+};
+
+export const getShellClient = () => {
+  return shellClient;
+};
+
+export default { setShellClient, getShellClient };
```

Comment:
Yes it is the same file. I asked the question at a daily and for now, we will duplicate it. As for example, hub has the sidebar visible by default and this can be setted in this file. This rule is only applied in hub and not in other apps.